osf/migrations/0215_auto_20200831_1635.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-08-31 16:35
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0214_auto_20200701_1658'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='abstractnode',
+            name='custom_storage_usage_limit_private',
+            field=models.PositiveIntegerField(blank=True, null=True),
```

Comment:
If we can have these be non-integer number field, it will be better, because (see below) I'd like to have admins enter the size in GB rather than the size in bytes.